lib/ansible/plugins/callback/__init__.py

```diff
@@ -41,6 +42,9 @@
 
 import yaml
 
+if typing.TYPE_CHECKING:
+    from ansible.executor.task_result import TaskResult
```

Comment:
this is not needed, the from future import of annotations takes care of this, also no need to make TaskResult a string below

Reply:
Sorry for the late response @bcoca, but I had to rebuild my dev machine. I removed the quotes around `TaskResult` since `from __future__ import annotations` removes the need for any forward-referencing hacks. But I’d like to keep @webknjaz suggestion to import only `if TYPE_CHECKING`.